app/assets/javascripts/discourse/tests/acceptance/sticky-avatars-test.js

```diff
@@ -1,28 +1,20 @@
-import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import { acceptance, query } from "discourse/tests/helpers/qunit-helpers";
 import { skip } from "qunit";
-import { find, scrollTo, visit, waitUntil } from "@ember/test-helpers";
-import { setupApplicationTest as EMBER_CLI_ENV } from "ember-qunit";
-
-acceptance("Sticky Avatars", function (needs) {
-  if (!EMBER_CLI_ENV) {
-    return; // helpers not available in legacy env
-  }
-
-  const container = document.getElementById("ember-testing-container");
-
-  needs.hooks.beforeEach(function () {
-    container.scrollTop = 0;
-  });
+import { visit } from "@ember/test-helpers";
 
+acceptance("Sticky Avatars", function () {
   skip("Adds sticky avatars when scrolling up", async function (assert) {
```

Comment:
This test is actually skipped :sweat_smile:

Reply:
Yup! I tried to fix it and un-skip it, but it didn't work out 🙂 At least it's in a better shape for the next person that want to take a stab at it. (though I suppose it's more likely this test will be deleted 😛)